README-quick-start.md

```diff
@@ -0,0 +1,70 @@
+# How to install CMS locally from scratch
+This document cherry picks some commands that can already be found in [README-short.md](https://github.com/office-for-students/wagtail-CMS/blob/develop/README-short.md)
+
+Notes:
+1. Python 3.6.8 is referenced in [Confluence](https://mobilisecloud.atlassian.net/wiki/spaces/OFS/pages/236912643/Managing+multiple+Python+versions) and [Docker](https://github.com/office-for-students/wagtail-CMS/blob/master/Dockerfile). If you don't already have 3.6 installed, a later version should work too.
+
```

Comment:
it doesn't haha. Best to stick with 3.6 as a new starter will certainly encounter issues.

Reply:
Siobhan doesn't have 3.6 so is using 3.7.2 or something - perhaps I should put "should" in italics :-)